src/chartsFactory.jsx

```diff
@@ -1,4 +1,5 @@
 var React = require('react');
+var win = global || window;
```

Comment:
Did you have a chance to test it?
If the code above doesn't work, i'd do something like this:
```javascript 
const win = typeof global == 'undefined' ? window : global;
```